data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/stream/BinlogEventListener.java

```diff
@@ -51,6 +54,7 @@ public class BinlogEventListener implements BinaryLogClient.EventListener {
     static final String CHANGE_EVENTS_PROCESSING_ERROR_COUNT = "changeEventsProcessingErrors";
     static final String BYTES_RECEIVED = "bytesReceived";
     static final String BYTES_PROCESSED = "bytesProcessed";
+    static final String REPLICATION_LOG_EVENT_PROCESSING_TIME = "replicationLogEventProcessingTime";
```

Comment:
Is there another name that would be appropriate instead of "event" here? This can be confusing with Data Prepper Events.

Reply:
Agreed. I can change it to "record", "item", or "entry" to differentiate. How about "replicationLogEntryProcessingTime"?

Reply:
I like that term - "entry".